specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/Labels.json

```diff
@@ -0,0 +1,361 @@
+          },
+          {
+            "name": "containerAppName",
+            "in": "path",
+            "description": "Name of the Container App.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-z][a-z0-9-]*$"
+          },
+          {
+            "name": "labelsName",
+            "in": "path",
+            "description": "Name of the Container App Labels collection.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-z][a-z0-9]*$"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Delete operation is in progress.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "204": {
+            "description": "Container App Labels does not exist."
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete Container App Labels": {
+            "$ref": "./examples/Labels_Delete.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    }
+  },
+  "definitions": {
+    "Labels": {
+      "description": "Container App Labels.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Labels specific properties",
+          "type": "object",
+          "properties": {
+            "provisioningState": {
+              "description": "Current provisioning State of the operation",
+              "enum": [
+                "InProgress",
+                "Succeeded",
+                "Failed",
+                "Canceled"
+              ],
+              "type": "string",
+              "readOnly": true,
+              "x-ms-enum": {
+                "name": "LabelsProvisioningState",
+                "modelAsString": true
+              }
+            },
+            "labels": {
+              "description": "Labels for the Container App.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Label"
+              },
+              "x-ms-identifiers": [
+                "name"
+              ]
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "Label": {
+      "description": "Label for the Container App.",
+      "type": "object",
+      "properties": {
+        "name": {
```

Comment:
Top level property names should not be repeated inside the properties bag for ARM resource.
https://github.com/Azure/azure-openapi-validator/blob/main/docs/arm-resource-properties-bag.md